### ISSUE TITLE
The bot should stop when it can't cast a certain number of fishing rods.

### ISSUE THREAD

**Issue by tahakuzgun** [cobra-7777/new-world-fish-bot#5]
The bot should stop when it can't cast a certain number of fishing rods.

**Comment by cobra-7777** [comment]
Im unsure what you mean, can you elaborate? 
A certain number of fishing rods? 
You should only be fishing with one fishing rod

**Comment by tahakuzgun** [comment]
Oh sorry i typed wrong.
For example:
The character threw a fishing rod 5 times, but failed. The bot should then stop itself.
It is much needed because of the problem I wrote in another topic. 
Character tries to throw a stone or the ground after 3-4 hours. Can't catch any fish. And throws a fishing rod on the ground. for hours.
A very bad view from the outside /and risky

**Comment by cobra-7777** [comment]
Why does this happen in the first place? Are you not repairing the rod often enough?
I could make the bot stop after not catching fish for a while, but you shouldnt be botting in populated areas anyway. Ill consider implementing it though

**Comment by tahakuzgun** [comment]
Yes, I don't do it in pop places. But some people can take you to a different place slowly with Ice Pylon. It may very well be that it shuts down when it fails more than once for security purposes.

https://www.youtube.com/watch?v=T1Q3yffHgNg
